### PR TITLE
Updating redirects for support page moves

### PR DESCRIPTION
**Pull request by virginiaulrich**

### CODE REVIEW

**Comment by virginiaulrich** [comment]
@jeanduplessis do you mind making sure I truly did this correctly? I noticed looking at the list that a lot of things didn't end in the `/`, so I left that off ... but perhaps that was wrong? Thank you for teaching me how to do this!

File: _resources/assets/redirects [hunk 1]
```diff
@@ -41,4 +41,5 @@
 /handbook/product/personas /handbook/marketing/personas 308
 /handbook/values /company/values 308
 
-
+/handbook/ce/support /handbook/support 308
+/handbook/ce/support/#slas /handbook/support#our-service-level-agreements-slas 308
```
Comment (jeanduplessis) [inline _resources/assets/redirects:45]:
```suggestion

```
This rule isn't needed/won't work due to the fragment (# symbol and stuff behind it) never reaching the backend, only the client (browser) knows about it.

Comment (jeanduplessis) [reply _resources/assets/redirects:45]: You could add an additional anchor to the SLAs section on the support page to catch the old fragment.

Comment (virginiaulrich) [reply _resources/assets/redirects:45]: Ah got it. Will update that page, then, too. Thank you!

**Pull request merged**